### PR TITLE
chore: Bump edgelb version

### PR DESCRIPTION
**Pull request by sreis**
## Description
* Track edgelb master. Contains a breaking change related to CloudProvider configuration. 
* Update release notes

When this is merged we can tag v0.1.0-alpha.12 to share with the customer.

### CODE REVIEW

**Comment by sreis** [comment]
Waiting on more information to update the release notes.

File: RELEASE-NOTES.adoc [hunk 1]
```diff
@@ -19,6 +19,16 @@ endif::[]
 == Table of Contents
 toc::[]
 
+== v0.1.0-alpha.12
+
+=== Breaking changes
+
+* To use NLB functionality we now require EdgeLB built from master (commit https://github.com/mesosphere/dcos-edge-lb/commit/d3547ca23ba9b4a07b91e33562868b731330c855[d3547ca]) or version 1.4.0.
```
Comment (jimmidyson) [inline RELEASE-NOTES.adoc]: This should explain what the breaking change is.

File: docs/usage/00-installing.adoc [hunk 2]
```diff
@@ -27,7 +27,7 @@ To install `dklb`, the following prerequisites must be met:
 ** DC/OS must be v1.12.0 or later.
 ** EdgeLB must be built and installed from the https://github.com/mesosphere/dcos-edge-lb/tree/master[`master`] branch.
    A stub-universe is available, see https://github.com/mesosphere/dcos-edge-lb#stub-universe[here] for details.
-*** EdgeLB https://docs.mesosphere.com/services/edge-lb/1.3/[`v1.3.0`] may be used instead, but some functionality will be missing.
+*** EdgeLB https://docs.mesosphere.com/services/edge-lb/1.4/[`v1.4.0`] may be used instead, but some functionality will be missing.
```
Comment (jimmidyson) [inline docs/usage/00-installing.adoc]: Is there functionality missing in 1.4.0? Why use it then?

File: go.mod [hunk 3]
```diff
@@ -22,7 +22,7 @@ require (
 	github.com/imdario/mergo v0.3.6 // indirect
 	github.com/json-iterator/go v1.1.5 // indirect
 	github.com/kr/pretty v0.1.0 // indirect
-	github.com/mesosphere/dcos-edge-lb v1.3.2-0.20190618110118-50a400af8d6f
+	github.com/mesosphere/dcos-edge-lb v1.3.2-0.20190628160932-d3547ca23ba9
```
Comment (jimmidyson) [inline go.mod]: Is there no 1.4.0 tag?

Comment (sreis) [reply go.mod]: It's not released yet.

File: docs/usage/00-installing.adoc (hunk 2 above)
Comment (sreis) [reply docs/usage/00-installing.adoc]: I think the original intent was to say that you should use `master`. What do you think of changing to `but some functionality might be missing`?

File: RELEASE-NOTES.adoc (hunk 1 above)
Comment (sreis) [reply RELEASE-NOTES.adoc]: Done, lmk what you think.

**Pull request merged**